### ISSUE TITLE
Add support for GitHub Actions

### ISSUE THREAD

**Issue by j-f1** [watson/ci-info#31]
**Details:**
 - Ci name: GitHub Actions
 - Website: https://github.com/features/actions
 - Default env variables: https://developer.github.com/actions/creating-github-actions/accessing-the-runtime-environment/#environment-variables

**Environment variables**

<details>
<summary>Environment variables example</summary> 

```
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=fb1d0264fdb4
GITHUB_ACTION=Env
GITHUB_ACTOR=j-f1
GITHUB_EVENT_NAME=push
GITHUB_EVENT_PATH=/github/workflow/event.json
GITHUB_REPOSITORY=j-f1/eslint-docs
GITHUB_SHA=50141211ee907a2f592f03d99caf45c131fb6782
GITHUB_WORKFLOW=Run tests
GITHUB_WORKSPACE=/github/workspace
HOME=/github/home
GITHUB_REF=refs/heads/tmp
```

</details>

**Additional context**
It would be nice to add support for this before it gets out of beta.

**Comment by fishcharlie** [comment]
Especially since GitHub Actions added support for CI/CD this would be especially helpful. @watson

**Comment by sibiraj-s** [comment]
Fixed in #42
